examples/settings.rs

```diff
@@ -1,14 +1,15 @@
 use meilisearch_sdk::client::Client;
 use meilisearch_sdk::indexes::Index;
+use meilisearch_sdk::request::IsahcClient;
 use meilisearch_sdk::settings::Settings;
 
 // we need an async runtime
 #[tokio::main(flavor = "current_thread")]
 async fn main() {
-    let client: Client = Client::new("http://localhost:7700", Some("masterKey"));
+    let client: Client<IsahcClient> = Client::new("http://localhost:7700", Some("masterKey"));
 
     // We try to create an index called `movies` with a primary_key of `movie_id`.
-    let my_index: Index = client
+    let my_index: Index<IsahcClient> = client
```

Comment:
This is a huge breaking and impacts the user experience considerably :( 

Can we have the `isahcClient` by default which you only would have to declare if you are in `no default feature` mode?

Reply:
You are right. Thanks for checking and taking the time to help me out 😊
This was a destructive change for users who had specified the type :(

I modified the SDK so that users who use it by default can use it without worrying about traits.(but a little forced😢)
However, I'm not sure if this is a good thing to do, as it also requires changes to the export and to `lib.rs`. 

All in all, this is a big change 
so I hope the maintainer team will consider the decision and any better response 🙏

https://github.com/meilisearch/meilisearch-rust/pull/459/commits/6d96c9f956b6ef02e6a92b41b017d5a06eb840e0

Reply:
What do you think @irevoire ?

Reply:
Yeah, I don't see any other way to do it tbh.
What would you think of having a `prelude` like they have in some libs @bidoubiwa?
For example, `rayon` uses one (look at their examples. They all starts with `use rayon::prelude::*;` instead of picking each thing you need separately: https://docs.rs/rayon/latest/rayon/iter/trait.ParallelIterator.html#method.for_each_with
serenity also exposes a prelude: https://docs.rs/serenity/latest/serenity/client/struct.Client.html

I feel like that's the easiest way for us to define all the types we need to overwrite in a prelude file gated by the feature being used.

That being said, it's a « big » breaking change, it'll probably break a lot of code, but the fix for the end users will be way easier => get rids of all your imports and just write `use meilisearch_sdk::prelude::*;`
The cool thing for us is that we can easily change the type alias in the prelude file without being breaking in the future + create a prelude for the wasm + a prelude for isahc

Reply:
I tried to modify the form to use `prelude`!
Please let me know if I'm wrong🙏

https://github.com/meilisearch/meilisearch-rust/pull/459/commits/bd04a9bd5be75cdd03b43b0dc0ed92b4c6bbe199
https://github.com/meilisearch/meilisearch-rust/pull/459/commits/5b326278139e521261c659a1a0e83a84890cb1ae